issues-workflow.md

```diff
@@ -40,12 +41,15 @@ There are five different stages of the triage:
 | Validity | Asses the validity of the issue (whether it is taken for the further triage and proper classification). | invalid, duplicate, wontfix, question |
 | Kind | Differentiate whether the related issue is a new feature or a bug. | enhancement, bug |
 | Help | Identify issues that do not have high priority and can be taken by the community. | help wanted, good first issue|
-| Priority | Prioritize issues in the general Kyma backlog to select those which are the most critical and should be taken as first. | critical |
+| Security | Specify CVSSv3 security rating https://www.first.org/cvss/calculator/3.0 with support of security team | security/{SCORE} |
+| Priority | Prioritize issues in the general Kyma backlog to select those which are the most critical and should be taken as first. | priority/critical |
 | Area | Clarify which capabilities are involved in a given issue. | area/{CAPABILITY_NAME} |
 
 ## Backlog
 
-The Kyma backlog contains issues that went through the triage, are not closed, and have labels added (except from the issues with the `question` label). Backlog prioritization is realized by assigning issues to Kyma milestones (ZenHub Release) and assigning the `critical` label. Critical issues assigned to the current milestone have the highest priority.
+The Kyma backlog contains issues that went through the triage, are not closed, and have labels added (except from the issues with the `question` label). Backlog prioritization is realized by assigning issues to Kyma milestones (ZenHub Release) and assigning the `priority/critical` label. Critical issues assigned to the current milestone have the highest priority.
+
+The Kyma backlog contains issues that went through the triage, are not closed, and have labels added (except from the issues with the `question` label). Backlog prioritization is realized by assigning issues to Kyma milestones (ZenHub Release) and assigning the `priority/critical` label. Critical issues assigned to the current milestone have the highest priorit
 
```

Comment:
Copy "paste past(e)" ? :) ...and by the way, it should be `except for` instead of `except from`.